app/modules/questionnaire/server.R

```diff
@@ -1,6 +1,8 @@
 source('app/modules/questionnaire/generate_questions_UI.R')
 source('app/modules/questionnaire/generate_recommendation.R')
 
+file.exists("app/data/questionnaire_inputs/q_database.csv")
+
 questions_df <- read.csv("app/data/questionnaire_inputs/q_database.csv")
```

Comment:
However this line is still troublesome as the app doesn't know where this path lies. You would need `here` package to do this. 
```suggestion
here::i_am('app/modules/questionnaire/server.R') #this defines the root directory of the project
data_directory <- here('app','data','questionnaire_inputs') #define relative path to your questionnaire app directory
questions_df <- read.csv(here(data_directory,"q_database.csv"))
```

Reply:
Even when I manually install here package it doesn't recognise the function.

<img width="1385" height="629" alt="image" src="https://github.com/user-attachments/assets/f2c8d2d3-b1e0-4f82-baad-e960478a2cf7" />

<img width="1086" height="219" alt="image" src="https://github.com/user-attachments/assets/91978162-0cf1-44f0-8a1f-c9e46afe2464" />